### PR TITLE
OvmfPkg: document runtime config options

### PR DESCRIPTION
**Pull request by kraxel**
Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>

### CODE REVIEW

File: OvmfPkg/RUNTIME_CONFIG.md [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+  --add-cert ${GUID} /etc/pki/tls/certs/${server1}.crt \
+  --add-cert ${GUID} /etc/pki/tls/certs/${server2}.crt
+qemu-system-x86_64 \
+  -fw_cfg name=etc/edk2/https/cacerts,file=mycerts.esl
+```
+
+
+## TLS: etc/edk2/https/ciphers
+
+Configres the allowed TLS chiper suites.  Using the hosts system
```
Comment (lersek) [inline OvmfPkg/RUNTIME_CONFIG.md:41]: host's

File: OvmfPkg/RUNTIME_CONFIG.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+qemu-system-x86_64 -fw_cfg name=opt/org.tianocore/X-Cpuhp-Bugcheck-Override,string=yes
+```
+
+
+## Platform: opt/ovmf/X-PciMmio64Mb
+
+Historically OVMF has by default used 32GB of address space for the
+64-bit PCI MMIO window.  This option can be used to change that size.
+
+Recent OVMF versions have started use an larger MMIO window by
```
Comment (lersek) [inline OvmfPkg/RUNTIME_CONFIG.md:136]: - "started using" or "started to use"
- "a larger"

File: OvmfPkg/RUNTIME_CONFIG.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+```
+qemu-system-x86_64 -fw_cfg name=opt/ovmf/X-PciMmio64Mb,string=131072
+```
+
+
+## Platform: physical address space bits
+
+OVMF will check the supported physical address space bits (via CPUID)
+when creating the address space layout.  OVMF will use the complete
+available address space and place 64-bit PCI MMIO window near the top
```
Comment (lersek) [inline OvmfPkg/RUNTIME_CONFIG.md:153]: "place **the** 64-bit PCI MMIO window"

**Review by lersek — APPROVED** [review]
Very nice! Pointed out a few typos, but those aren't deal-breakers.

File: OvmfPkg/RUNTIME_CONFIG.md (hunk 3 above)
Comment (kraxel) [reply OvmfPkg/RUNTIME_CONFIG.md]: Hi @lersek, nice to see you here again ;)
Thanks for proof-reading, fixed the typos.

Comment (lersek) [reply OvmfPkg/RUNTIME_CONFIG.md]: looks good, kudos

**Comment by kraxel** [comment]
Ping?
There are two pending PRs (https://github.com/tianocore/edk2/pull/6494 + https://github.com/tianocore/edk2/pull/6211) which add more options.  When merging this quickly the PRs can easily update RUNTIME_CONFIG.md directly.

**Review by ardbiesheuvel — APPROVED** [review]

**Pull request merged**